bundles/org.openhab.core.io.monitor/bnd.bnd

```diff
@@ -6,4 +6,6 @@ Import-Package: \
  org.osgi.service.*,\
  org.slf4j.*,\
  com.google.gson.*;version="[2.8,3)"
-Export-Package: io.micrometer.core.*
+Export-Package: io.micrometer.core.*;-split-package:=merge-first,\
```

Comment:
What is the split-package good for?

Reply:
I added them because there was a warning in the bnd-maven plugin:
`Split package, multiple jars provide the same package` - thb I don't know where these duplicates originate from, I'm quite inexperienced with bnd/OSGI. 
The split-package directive basically just sets the default strategy explicitly and thus removes the warning, not really the root cause. Any advice appreciated.

Reply:
> I don't know where these duplicates originate from

My assumption is that this is because the original jar is on the build path as well as the extracted classes - so bnd thinks that two different bundles provide this package. The setting should thus be useless at runtime. It only suppresses the warning at build time. Not sure if we should keep it for this.

@wborn As our bnd expert: Do you have any opinion here?